### ISSUE TITLE
Edit OTS Banner Map

### ISSUE THREAD

**Issue by mstevens-atx** [cityofaustin/atd-geospatial#106]
Need to edit the Over-the-Street Interactive Web Application to remove 11 as a valid location. It will be added to the list of inactive locations.

**Comment by mstevens-atx** [comment]
Added the text on the web app. Also, changed the USABLE field for 11, 49, and 65 to 'No' and set a filter in the map to only show USABLE is 'Yes'. The only thing left is to wait for it to auto-update everything from MAINT to AGOL.

**Comment by jaime-mckeown** [comment]
- I checked this map and the points were still drawing. I emailed Matt about the request for clarification. I realized that the layer just needed to have a filter added to turn those points off.
- Matt also pointed out that the Web App was acting strange when you clicked on a link on the left that the pop-up that was opening is incorrect. I showed Diana the problem, and she helped me figure out what I needed to do to fix the problem. I will need to re-do all of the links map location to fix the pop-ups.
